src/index.js

```diff
@@ -65,6 +66,9 @@ chaiJestSnapshot.resetSnapshotRegistry = function resetSnapshotRegistry() {
   }
 }
 
+chaiJestSnapshot.createState = snapshotStateHandler.create;
+chaiJestSnapshot.removeUncheckedKeys = snapshotStateHandler.removeUncheckedKeys;
```

Comment:
We will have to refactor the documentation to note that you need to call these in your `beforeAll`/`afterAll`.